acceptance/chaos_test.go

```diff
@@ -56,20 +64,55 @@ func TestChaos(t *testing.T) {
 	var stalled int32
 	clients := make([]struct {
 		sync.RWMutex
-		db      *client.DB
+		db      *sql.DB
 		stopper *stop.Stopper
 		count   int64
 	}, num)
 
 	// initClient requires that the caller holds the client's write lock.
 	initClient := func(i int) {
-		db, dbStopper := makeClient(t, c.ConnString(i))
+		db := makePGClient(t, c.PGUrl(i))
 		if clients[i].stopper != nil {
 			clients[i].stopper.Stop()
 		}
-		clients[i].db, clients[i].stopper = db, dbStopper
+		clients[i].db = db
+		clients[i].stopper = stop.NewStopper()
 	}
 
+	// Initialize the "accounts" table.
+	db := makePGClient(t, c.PGUrl(0))
+
+	if _, err := db.Exec(`CREATE DATABASE IF NOT EXISTS bank`); err != nil {
+		t.Fatal(err)
+	}
+
+	schema := `
+CREATE TABLE IF NOT EXISTS bank.accounts (
```

Comment:
It's better to use `DROP TABLE IF EXISTS bank.accounts` and then create it instead of `CREATE TABLE IF NOT EXISTS` followed by truncate. This way you know you're using the right schema instead of a different table that may have been left by a different test run.